recipes/iwyu/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+CC=clang
+CXX=clang++
```

Comment:
Shouldn't this be the default compilers?  Why is there a need to override them?

Reply:
no reason at all, normally I prefer to use clang over gcc, but actually there is no specific reason for that.